docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,72 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+When jobs exceed the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). To avoid OOM error, you can increase the **memory** limits per workflow step to make an efficient use of the available resources. Note that you can also decrease the memory limit to increase the chances of jobs being scheduled earlier if the cluster is busy.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit` in _**each step specification**_.
+
+You can read more about the expected memory values on [Kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
```

Comment:
```suggestion
Read more about the expected memory values on [Kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
```
To avoid "You can..." repetition